index.html

```diff
@@ -289,7 +289,7 @@ <h2 class="stt">LET’S CONNECT</h2>
         <li>
           <div id="spec_name">
             <p class="lfs">Skills</p>
-            <div class="img0"><img src="statics/images/select.png" alt="select"></div>
+            <div class="img0"><img src="statics/images/select.png" alt="select skills"></div>
           </div>
           <div class="lhr" id="l2">
```

Comment:
[Optional] I think that it'll be useful to use more meaningful class names that'll help understand the context rather than just a random combination of text.

Reply:
Thanks for the tips. I changed in some places and will maintain it in upcoming projects. Thanks